src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -896,6 +896,48 @@ void jit_select_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const
     h->mov(dst.b16, aux.b16);
 }
 
+// SoftPlus
+jit_soft_plus_emitter::jit_soft_plus_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
+                                             dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                             const std::shared_ptr<ov::Node>& node)
+                                             : jit_emitter(host, host_isa, node, get_arithmetic_binary_exec_precision(node)) {
+}
+jit_soft_plus_emitter::jit_soft_plus_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
+                                             dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                             const ov::element::Type exec_prc)
+                                             : jit_emitter(host, host_isa, exec_prc) {
+}
+size_t jit_soft_plus_emitter::get_inputs_count() const { return 1; }
+size_t jit_soft_plus_emitter::get_aux_vecs_count() const { return 1; }
+
+std::set<std::vector<element::Type>> jit_soft_plus_emitter::get_supported_precisions(const std::shared_ptr<ov::Node>& node) {
+    return {{element::f32}};
+}
+
+void jit_soft_plus_emitter::emit_impl(const std::vector<size_t>& in_vec_idxs, const std::vector<size_t>& out_vec_idxs) const {
+    if (host_isa_ == dnnl::impl::cpu::aarch64::asimd) {
+        emit_isa<dnnl::impl::cpu::aarch64::asimd>(in_vec_idxs, out_vec_idxs);
+    } else {
+        OV_CPU_JIT_EMITTER_THROW("Can't create jit eltwise kernel");
+    }
+}
+
+template <dnnl::impl::cpu::aarch64::cpu_isa_t isa>
+    void jit_soft_plus_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+        OV_CPU_JIT_EMITTER_ASSERT(exec_prc_ == ov::element::f32, "unsupported precision: " + exec_prc_.to_string());
+        using TReg = typename dnnl::impl::cpu::aarch64::cpu_isa_traits<isa>::TReg;
+        const TReg src = TReg(in_vec_idxs[0]);
+        const TReg dst = TReg(out_vec_idxs[0]);
+        const TReg aux = TReg(aux_vec_idxs[0]);
+        h->movi(aux.s, 1);
+        h->fadd(dst.s, src.s, 1);
+        h->fdiv(dst.s, dst.s, 1);
+        h->fsub(dst.s, 1, dst.s);
+        h->fmul(dst.s, dst.s, src.s);
```

Comment:
`SoftPlus` implementation: https://docs.openvino.ai/2024/documentation/openvino-ir-format/operation-sets/operation-specs/activation/softplus-4.html
You can use reference for `x64` from `oneDNN`: https://github.com/openvinotoolkit/oneDNN/blob/6bf3e2356258149cedf4e31a2c41b9105b4a0da0/src/cpu/x64/injectors/jit_uni_eltwise_injector.cpp#L667